src/graphql/node/crud.rs

```diff
@@ -395,32 +212,17 @@ mod tests {
                         customerId: 0,
                         description: "This is the admin node running review.",
                         hostname: "admin.aice-security.com",
-                        piglet: false,
-                        pigletGigantoIp: null,
-                        pigletGigantoPort: null,
-                        savePackets: false,
-                        http: false,
-                        office: false,
-                        exe: false,
-                        pdf: false,
-                        txt: false,
-                        vbs: false,
-                        smtpEml: false,
-                        ftp: false,
-                        giganto: false,
-                        gigantoIngestionIp: null,
-                        gigantoIngestionPort: null,
-                        gigantoPublishIp: null,
-                        gigantoPublishPort: null,
-                        gigantoGraphqlIp: null,
-                        gigantoGraphqlPort: null,
-                        retentionPeriod: null,
-                        reconverge: false,
-                        hog: false,
-                        hogGigantoIp: null,
-                        hogGigantoPort: null,
-                        protocols: null,
-                        sensors: null,
+                        agents: [{
+                            key: "reconverge_key"
```

Comment:
Since the format of this key is like "reconverge@analysis.mydomain.com", could you use something similar including the later ones?